testcases/kernel/security/stack_clash/.gitignore

```diff
@@ -0,0 +1 @@
+stack_clash
```

Comment:
I've pushed first batch of the CVE tests to the git, please rebase your patch on the top of that.